snippets/rust.json

```diff
@@ -20,6 +20,10 @@
       "prefix": "for",
       "body": "for ${1:pat} in ${2:expr} {\n\t$3\n}"
     },
+    "format": {
+      "prefix": "format",
+      "body": "format!(\"${1:\\{\\}}\", ${2:value})$0"
```

Comment:
`$0` is always the last tabs stop and therefore quite suitable for this use case. But if you prefer the direkt from with `$3`, I will change it.